src/ServicePulse.Host/app/js/directives/ui.particular.graph.js

```diff
@@ -73,9 +73,14 @@
                             .attr("font-family", "sans-serif")
                             .text(average.toFixed(2));
 
+                        var scaleYValue = scaleY(average == 0 ? 1 : average) + 3;
+                        if (isNaN(scaleYValue)) {
```

Comment:
Do we want to replace it with `scaleYValue | 0`. It's cosmetic so I can live with isNaN